serverless/installing_serverless/installing-knative-eventing.adoc

```diff
@@ -4,50 +4,22 @@ include::modules/serverless-document-attributes.adoc[]
 include::modules/common-attributes.adoc[]
 :context: installing-knative-eventing
 
-You must create a `KnativeEventing` object to install Knative Eventing using the {ServerlessOperatorName}.
+toc::[]
 
-[IMPORTANT]
-====
-You must create the `KnativeEventing` object in the `knative-eventing` namespace, as shown in the sample YAML, or the object is ignored.
-====
+Knative Eventing on {product-title} enables developers to more easily declare how components of their system communicate, using an event-driven architecture for serverless applications.
+Event-driven architecture is based on the concept of decoupled relationships between event producers and event consumers.
 
-[source,yaml]
-.Sample `eventing.yaml`
-----
-apiVersion: v1
-kind: Namespace
-metadata:
-  name: knative-eventing
----
-apiVersion: operator.knative.dev/v1alpha1
-kind: KnativeEventing
-metadata:
-  name: knative-eventing
-  namespace: knative-eventing
-----
+For more information about event-driven architecture, see link:https://www.redhat.com/en/topics/integration/what-is-event-driven-architecture[What is event-driven architecture?]
 
+// Create Eventing project + namespace
+include::modules/serverless-create-eventing-namespace.adoc[leveloffset=+1]
 
-.Prerequisite
-* An account with cluster administrator access.
+== Installing Knative Eventing
+
+.Prerequisites
+* An {product-title} account with cluster administrator access.
```

Comment:
Hit and run comment here, but fragments in an UL shouldn't end with punctuation (ISG ch. 3, _Punctuation in lists_).